packages/patternfly-4/react-core/src/components/Chip/Chip.d.ts

```diff
@@ -0,0 +1,20 @@
+import { SFC, HTMLProps } from 'react';
+import { Omit, OneOf } from '../../typeUtils';
+import { TooltipPosition } from '../Tooltip';
+
+export const ChipVariant = {
+  overflow: 'overflow',
+  closable: 'closable'
+};
+
+export interface ChipProps extends Omit<HTMLProps<HTMLDivElement>, 'children' > {
+  children: string;
```

Comment:
Can you get rid of the Omit 'children' and remove children from the interface? Also don't need id and onClick I believe since they are included in HTMLProps